src/components/Activity/ActivityItem.js

```diff
@@ -12,23 +12,25 @@ import { network } from '../../environment'
 import { shortenAddress, transformAddresses } from '../../web3-utils'
 import { activityStatusTypes } from '../../contexts/ActivityContext'
 import AppIcon from '../AppIcon/AppIcon'
-import TimeTag from './TimeTag'
 import IconSuccess from '../../icons/IconSuccess'
 import IconPending from '../../icons/IconPending'
+import TimeTag from './TimeTag'
+import TransactionProgress from './TransactionProgress'
 
 const ActivityItem = ({ activity, onClose }) => {
   const { app } = activity
   return (
     <section
       css={`
         display: grid;
+        align-items: center;
         grid-template-areas:
           'title time'
           'content content';
         overflow: hidden;
         position: relative;
         width: 100%;
-        padding: 20px;
+        padding: 24px;
```

Comment:
Pierre introduced a first version of [grid units](https://github.com/aragon/aragon/blob/master/src/utils.js#L111) and I liked using it in one of my PRs, would it make sense for you to start trying it out too (here)?

Reply:
Yes!